release-tools/prow.sh

```diff
@@ -86,7 +86,7 @@ configvar CSI_PROW_BUILD_PLATFORMS "linux amd64 amd64; linux ppc64le ppc64le -pp
 # which is disabled with GOFLAGS=-mod=vendor).
 configvar GOFLAGS_VENDOR "$( [ -d vendor ] && echo '-mod=vendor' )" "Go flags for using the vendor directory"
 
-configvar CSI_PROW_GO_VERSION_BUILD "1.22.5" "Go version for building the component" # depends on component's source code
+configvar CSI_PROW_GO_VERSION_BUILD "1.22.7" "Go version for building the component" # depends on component's source code
```

Comment:
release-tools changes have to be made in csi-release-tools repo first and use git subtree to import the changes to the sidecar repo. See instructions here: https://github.com/kubernetes-csi/csi-release-tools/blob/master/README.md#sharing-and-updating

Reply:
ok, i will revert it. https://github.com/kubernetes-csi/csi-release-tools/pull/261 is already taking care to update the go version in release-tools